common/tools/dev-tool/register.js

```diff
@@ -13,22 +13,16 @@
  * old JavaScript, it is simple enough.
  */
 
-const path = require("path");
+const { existsSync } = require("node:fs");
```

Comment:
Thanks for the feedback! I am not opposed to it but will have to see what that changes (since this would require top-level await or wrapping the bootstrap as async). For now I think it's fine to stay synchronous but if we run into performance issues we'll need to investigate 👍